scripts/modules/b2b-account/edit-quote.js

```diff
@@ -1128,6 +1134,69 @@ define([
                 }
             }
             return false;
+        },
+        applyCoupon: function () {
+            var couponCode = $('input[data-mz-value=couponCode]').val();
+            if (couponCode) {
+                var self = this;
+                var data = {
+                    quoteId: self.model.get('id'),
+                    couponCode: couponCode,
+                    updateMode: applyToDraft
+                };
+                self.model.isLoading(true);
+                self.model.apiModel.applyCoupon(data).then(function (response) {
+                    self.resetModel(response.data);
+                    var couponIsNotApplied = _.find(response.data.invalidCoupons, function (d) {
+                        return d.couponCode && d.couponCode === couponCode;
+                    });
+
+                    if (couponIsNotApplied) {
+                        var error = { message: couponCode + " " + couponIsNotApplied.reason };
+                        self.showMessageBar(error);
+                    }
+                }, function (error) {
+                    self.showMessageBar(error.items[0]);
```

Comment:
If we have multiple error messages then, create one error message by looping on items and show it on UI. You can use "<br />" tag to separate the messages.

Reply:
Actually here we are getting an error in the array that what added error.items[0]